installing/installing_oci/installing-oci-agent-based-installer.adoc

```diff
@@ -10,6 +10,11 @@ In {product-title} {product-version}, you can use the Agent-based Installer to i
 
 Installing a cluster on {oci} is supported for virtual machines (VMs) and bare-metal machines.
 
+[NOTE]
+====
+You can deploy {product-title} on a link:https://www.oracle.com/uk/cloud/cloud-at-customer/dedicated-region/[Dedicated Region] the same as any region from {oci-first-no-rt}.
```

Comment:
Just checking if `(Oracle documentation)` can be added after "....[Dedicated Region]"

Reply:
I thought about it an initially didn't think it would be necessary, but I think the docs guidelines indicate that I should put something after the link here